src/ddmd/backend/debug.c

```diff
@@ -318,25 +319,34 @@ void WRblock(block *b)
         int ncases;
 
         assert(b);
-        printf("***** block %p ", b);
-        WRBC(b->BC);
+        printf("%2d: ", b->Bnumber); WRBC(b->BC);
         if (b->Btry)
-            printf(" Btry=%p",b->Btry);
+            printf(" Btry=B%d",b->Btry ? b->Btry->Bnumber : 0);
         if (b->Bindex)
             printf(" Bindex=%d",b->Bindex);
         if (b->BC == BC_finally)
-            printf(" b_ret=%p", b->BS.BI_FINALLY.b_ret);
+            printf(" b_ret=B%d", b->BS.BI_FINALLY.b_ret ? b->BS.BI_FINALLY.b_ret->Bnumber : 0);
 #if MARS
         if (b->Bsrcpos.Sfilename)
             printf(" %s(%u)", b->Bsrcpos.Sfilename, b->Bsrcpos.Slinnum);
 #endif
         printf("\n");
-        if (b->Belem) elem_print(b->Belem);
+        if (b->Belem)
+        {
+            if (0)
```

Comment:
Should this be `false` instead of `0`?

Reply:
The reason I ask is because of #7310

Reply:
`0` is never going to be an incorrect argument to `if`.